draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -117,31 +110,31 @@ To perform client authentication using this scheme, the client instance uses the
 
 The value of the "client_assertion_type" parameter (as defined in {{RFC7521}}) set to "urn:ietf:params:oauth:client-assertion-type:jwt-client-attestation".
 
-The value of the "client_assertion" parameter (as defined in {{RFC7521}}) set to a value containing two JWTs, separated by a '~' character. It MUST NOT contain more or less than precisely two JWTs separated by the '~' character. The first JWT MUST be the client attestation JWT defined in [](#client-attestation-jwt), the second JWT MUST be the client attestation PoP defined in [](#client-attestation-pop-jwt).
+The value of the "client_assertion" parameter contains a single JWT known as the client attestation JWT as defined defined in [](#client-attestation-jwt). It MUST NOT contain more than one JWT.
```

Comment:
I don't think the second sentence is necessary since it is implied by the former sentence.
```suggestion
The value of the "client_assertion" parameter contains a single JWT known as the client attestation JWT as defined defined in [](#client-attestation-jwt).
```

Reply:
I think we would want this constraint wouldn't we? Why would you like to remove it?

Reply:
Didn't read your original comment just the suggestion, while I agree I think this makes the normative requirement a little more explicit. I took the language structure from [RFC7523](https://datatracker.ietf.org/doc/html/rfc7523#section-2.2)

> The value of the "client_assertion" parameter contains a single JWT.
   It MUST NOT contain more than one JWT.

I'm ok with removing this additional sentence though.